libkernelflinger/android.c

```diff
@@ -1358,6 +1363,19 @@ static EFI_STATUS setup_command_line(
                         goto out;
         }
 
+#ifdef USE_SBL
+        /* Append cmd_for_kernel */
+        cmd_for_kernel = get_cmd_for_kernel();
+        tmp = (char *)cmd_for_kernel;
+        while ((tmp = strchr(tmp, '\\')) != NULL) {
+                *tmp = ' ';
+                tmp++;
+        }
+        cmd_for_kernel_l = stra_to_str((CHAR8 *)cmd_for_kernel);
+        ret = prepend_command_line(&cmdline16, L"%s", cmd_for_kernel_l);
```

Comment:
`%a` is the format string for ascii string in kernelflinger. Converting it to UTF16 string is unnecessary here.